src/gwt/src/org/rstudio/studio/client/workbench/views/output/find/FindInFilesDialog.java

```diff
@@ -58,15 +62,15 @@ public static native State createState(String query,
                                              String path,
                                              boolean regex,
                                              boolean caseSensitive,
-                                             boolean gitIgnore,
-                                             JsArrayString filePatterns) /*-{
+                                             JsArrayString filePatterns,
```

Comment:
Nit: could you rename this `includeFilePatterns` to mirror `excludeFilePatterns`, as well as other occurrences? I think that would make it a bit clearer when just glancing over the code.